### ISSUE TITLE
dose not work in NodeJS v20

### ISSUE THREAD

**Issue by jimmywarting** [za-creature/cf-emu#2]
seems to use global fetch when it's available instead of node-fetch which breaks a bunch of stuff.

Should just rip out all formdata/fetch/busboy stuff and just depend on using built in fetch in nodejs instead.
require that user use node v18+

create the vm context based on something like: 

```js
class WorkerScript extends EventTarget { ... }
```
and make a `FetchEvent` class based on 
```js
class FetchEvent extends Event { ... }
```

**Comment by za-creature** [comment]
I got all tests passing in node v18 using native `fetch` however before I prep a new release I want to make sure that it still covers my workflows. Most issues were caused by [undici](https://github.com/nodejs/undici) defaulting to `connection: keep-alive` which coupled with the default 5 second allotment caused most of the tests to time out. The rest were primarily due to my lax adherence to request / response immutability and some awkwardness involving the new `duplex` flag (which I've yet to test Cloudflare's conformance to).

I wrote this some years ago back when there was no alternative available and I'm still maintaining it as a public thank you to Cloudflare for hosting my websites for free and inviting me to attend one of their events a while back.

That being said, nowadays there are some (probably better) alternatives that you should consider instead: The first and foremost is cloudflare's very own [wrangler](https://github.com/cloudflare/workers-sdk) which is likely the authoritative source whenever the spec is unclear. If that doesn't cover your use case, you may consider [miniflare](https://github.com/cloudflare/miniflare) which is now... also sponsored by cloudflare? (sorry, this is news to me)

As for getting rid of busboy & co, I'm not quite ready to do that just yet. While undici is arguably a better implementation than the ragtag band of misfits I collected in this package, it's still non-conforming to some (documented) degree, in some cases because the WHATWG spec itself is odd in some places. I'm not against it, but ultimately I maintain this for myself and as long as it works, I don't necessarily care too much that it's not using the latest and greatest dependencies, especially when I see stuff like this in the logs:

`(node:4875) ExperimentalWarning: buffer.File is an experimental feature and might change at any time`

Keeping this open until the next release which will target recent LTS versions of node as well

**Comment by za-creature** [comment]
Fixed in [1.0.3](https://github.com/za-creature/cf-emu/compare/d04d17e...6830490)
